views/plugin/kml/kmlnodelayerui.html

```diff
@@ -19,6 +19,8 @@
           line-dash="lineDash"
           shape="shape"
           shapes="shapes"
+          allow-ellipse-config="false"
+          layer-nodes="vector.layerNodes"
```

Comment:
I originally was adding layerNodes because `vectorstylecontrols.js` already expects to have `layerNodes` as part of the scope. I think I can remove that entirely now though because above `layer-nodes` is `allow-ellipse-config="false"` so this ui is not supported/able to be used for kml node layers anyway.